proposals/lip-0048.md

```diff
@@ -31,18 +31,11 @@ Each chain can configure the token used to pay fees. On the Lisk mainchain, the
 
 ### Minimum Fee per Transaction
 
-As introduced in [LIP 0013][lip-0013], all transactions must have a fee greater or equal to a minimum fee (which can be zero). The minimum fee is computed from a part based on the transaction size and a part specific to the command (called _extra command fee_). Chains can configure their minimum fee per byte and the eventual extra fees.
-
-For example, on the Lisk mainchain, the following extra fees are defined:
-
-* `extraCommandFee(MODULE_ID_DPOS, COMMAND_ID_DELEGATE_REGISTRATION) = 1000000000`,
-* `extraCommandFee(MODULE_ID_INTEROPERABILITY, COMMAND_ID_SIDECHAIN_REG) = 1000000000`.
-
-The constants `MODULE_ID_DPOS` and `COMMAND_ID_DELEGATE_REGISTRATION` are defined in [LIP "Define state and state transitions of DPoS module"][research:dpos-module].  The constants `MODULE_ID_INTEROPERABILITY` and `COMMAND_ID_SIDECHAIN_REG` are defined in [LIP 0045][lip-0045].
+As introduced in [LIP 0013][lip-0013], all transactions must have a fee greater or equal to a minimum fee (which can be zero). The minimum fee is computed from the transaction size. Chains can configure their minimum fee per byte and the eventual extra fees.
 
 ### Burning the Minimum Fee
 
-The minimum part of the fee should be burned, this is to avoid that validators can send transactions in the blocks they generate without cost. This is only possible if the chosen fee token is a native token (with chain ID equal 0). If the chosen fee token is not a native token (for example the LSK token on sidechains), the minimum fee per byte and the extra fees should be set to zero. This is the only choice that makes sense, as burning non-native tokens is not supported by the [Token module][research:token-module].
+The minimum part of the fee should be burned, this is to avoid that validators can send transactions in the blocks they generate without cost. This is only possible if the chosen fee token is a native token (with chain ID equal 0). If the chosen fee token is not a native token (for example the LSK token on sidechains), the minimum fee per byte and the extra fees should be set to zero. This is the only choice that makes sense, as burning non-native tokens is not supported by the [Token module][token-module].
```

Comment:
```suggestion
The minimum part of the fee should be burned, this is to avoid that validators can send transactions in the blocks they generate without cost. This is only possible if the chosen fee token is a native token (with chain ID equal 0). If the chosen fee token is not a native token (for example the LSK token on sidechains), the minimum fee per byte and the extra fees should be set to zero. This is the only choice that makes sense, as burning non-native tokens is not supported by the [Token module][lip-0051].
```